### PR TITLE
Organize Application component

### PR DESCRIPTION
**Pull request by xinhaoz**

### CODE REVIEW

File: src/Components/Application/reviewReducer.js [hunk 1]
```diff
@@ -22,14 +22,10 @@ import { produce } from "immer";
...
 export const INSERT_COMMENT = "INSERT_COMMENT";
 export const UPDATE_RATING = "UPDATE_RATING";
-export const LOAD_REVIEW = "LOAD_REVIEW";
 
 const MASTER_ID = "master";
 
 export function reviewReducer(review, action) {
-  if (action.type === LOAD_REVIEW) {
-    return action.review;
-  }
```
Comment (ahmedhamodi) [inline src/Components/Application/reviewReducer.js:32]: Just confirming this was meant to be removed. Do we not have this action anymore?

**Review by ahmedhamodi — APPROVED** [review]
LGTM. One question - just ensuring this was meant to be removed  / won't break anything else if removed.

**Review by GodGreg — APPROVED** [review]
Xin I love you thankyou for doing this!

File: src/Components/Application/reviewReducer.js (hunk 1 above)
Comment (GodGreg) [reply src/Components/Application/reviewReducer.js:32]: Also wondering this. @xinhaoz

Comment (xinhaoz) [reply src/Components/Application/reviewReducer.js:32]: Yeah the only use case has been deleted.

**Pull request merged**